subprojects/language-native/src/main/java/org/gradle/language/swift/plugins/SwiftLibraryPlugin.java

```diff
@@ -91,45 +93,51 @@ public void apply(final Project project) {
         debugApiElements.extendsFrom(api);
         debugApiElements.setCanBeResolved(false);
         debugApiElements.getAttributes().attribute(Usage.USAGE_ATTRIBUTE, objectFactory.named(Usage.class, Usage.SWIFT_API));
-        debugApiElements.getAttributes().attribute(CppBinary.DEBUGGABLE_ATTRIBUTE, true);
+        debugApiElements.getAttributes().attribute(DEBUGGABLE_ATTRIBUTE, true);
```

Comment:
should really query the binary for these values.

Reply:
Done.